simple/components/webui/scripts/create.sh

```diff
@@ -46,4 +46,22 @@ ctx logger info "Deploying Grafana Configuration..."
 grafana_conf=$(ctx download-resource "components/webui/config/config.js")
 sudo mv ${grafana_conf} "${GRAFANA_HOME}/config.js"
 
+ctx logger info "Configuring logrotate..."
+lconf="/etc/logrotate.d/nodejs"
```

Comment:
This is quite like doing "/etc/logrotate.d/python". These are not nodejs logs but rather the webui's.